pyphare/pyphare/pharein/restarts.py

```diff
@@ -9,20 +9,24 @@
 
 def validate(sim):
 
-    sim.restart_options["timestamps"] = phare_utilities.np_array_ify(sim.restart_options["timestamps"])
-    init_time = sim.start_time()
-
-    timestamps = sim.restart_options["timestamps"]
-    if np.any(timestamps < init_time):
-        raise RuntimeError(f"Error: timestamp({sim.time_step_nbr}) cannot be less than simulation.init_time({init_time}))")
-    if np.any(timestamps > sim.final_time):
-        raise RuntimeError(f"Error: timestamp({sim.time_step_nbr}) cannot be greater than simulation.final_time({sim.final_time}))")
-    if not np.all(np.diff(timestamps) >= 0):
-        raise RuntimeError(f"Error: {clazz}.{key} not in ascending order)")
-    if not np.all(np.abs(timestamps / sim.time_step - np.rint(timestamps/sim.time_step) < 1e-9)):
-        raise RuntimeError(f"Error: {clazz}.{key} is inconsistent with simulation.time_step")
-
-    sim.restart_options["timestamps"] = conserve_existing(sim)
+    if "elapsed_timestamps" in sim.restart_options:
+        sim.restart_options["elapsed_timestamps"] = phare_utilities.np_array_ify(sim.restart_options["elapsed_timestamps"])
+
+    if "timestamps" in sim.restart_options:
+        sim.restart_options["timestamps"] = phare_utilities.np_array_ify(sim.restart_options["timestamps"])
```

Comment:
we should also probably decide in which time units the users provides their timestamps. Is that seconds, minutes, hours, days?
Also do they provide time stamps (i.e. dates) or real time intervals (seconds, hours, etc.)? 
I would think users provide timestamps **relative** to the start time, in some unit that we need to decide what it is.... I'd say hours?

Reply:
for the sake of testing it should probably support seconds

Reply:
maybe we can do something with "timedelta" / datetime https://docs.python.org/3/library/datetime.html ?

Reply:
yes I was considering this option too... @jeandet worked quite a bit on python/C++ time stuff, in case there are unexpected issues.